test/automated-suite.js

```diff
@@ -89,15 +91,31 @@ describe('Test SW-Toolbox', function() {
   automatedBrowsers.forEach(browserInfo => {
     // Tests are running differently from mocha compared to gulp test:manual
     // :( Results in errors that can't be debugged
-    if (browserInfo.getSeleniumBrowserId() === 'firefox' &&
+    if (process.env.TRAVIS && browserInfo.getSeleniumBrowserId() === 'firefox' &&
       browserInfo.getVersionNumber() <= 50) {
       console.log('Skipping ' + browserInfo.getRawVersionString());
       return;
     }
 
+    if (process.env.TRAVIS && browserInfo.getSeleniumBrowserId() === 'opera' &&
+      browserInfo.getVersionNumber() <= 39) {
+      console.log('Skipping ' + browserInfo.getRawVersionString());
+      return;
+    }
+
+    // This properly breaks the tests
+    if (browserInfo.getSeleniumBrowserId() === 'chrome' &&
+      browserInfo.getVersionNumber() === 54) {
```

Comment:
Need to wait for a new Chrome Driver: https://bugs.chromium.org/p/chromedriver/issues/detail?id=1473